browser/ui/webui/brave_wallet/wallet_common_ui.cc

```diff
@@ -13,9 +13,11 @@
 #include "brave/components/brave_wallet/browser/wallet_data_files_installer.h"
 #include "brave/components/constants/webui_url_constants.h"
 #include "chrome/browser/profiles/profile.h"
+#if !BUILDFLAG(IS_ANDROID)
 #include "chrome/browser/ui/browser.h"
 #include "chrome/browser/ui/browser_list.h"
 #include "chrome/browser/ui/tabs/tab_strip_model.h"
+#endif
 #include "chrome/browser/ui/webui/favicon_source.h"
```

Comment:
I'm not really sure how does build work, as that file isn't available on Android https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/BUILD.gn;l=1502?q=browser%2Fui%2FBUILD.gn&ss=chromium%2Fchromium%2Fsrc. Do you include deps to it somewhere?

Reply:
it is used by "brave/browser/ui/ui/brave_rewards_page_ui.cc"
https://github.com/brave/brave-core/blob/223bfff93b0f651eb9b83a363a77c33209f6c856/browser/ui/webui/brave_rewards_page_ui.cc#L314
and added long time ago

Reply:
I see we added it inside `//brave/browser/ui/BUILD.gn`. There is a note that it has to be removed eventually. Could you please check why do we need it on the wallet code for Android? I would avoid adding it if it has been added as a temp fix for rewards.

Reply:
It is used by rewards page:
https://github.com/brave/brave-core/blob/223bfff93b0f651eb9b83a363a77c33209f6c856/browser/ui/webui/brave_rewards_page_ui.cc#L314

Reply:
I moved it from the android